client/src/pages/reports/Form.js

```diff
@@ -536,11 +536,11 @@ const ReportForm = ({
                   }
                 >
                   {isFutureEngagement && (
-                    <HelpBlock>
+                    <Form.Text>
```

Comment:
You're using `FormText` from `react-bootstrap` in other places; which one has preference?